### PR TITLE
Package Install integration tests with multiple online targets

### PR DESCRIPTION
**Pull request by ct-ajovanovic**
Brief description of the PR.
Added second part of Package Install integration tests.

**Related Issue**
This PR fixes/closes part of #2662

**Description of the solution adopted**
This PR continues the work from PR #2733, and the mentioned PR should be merged first.
After that a rebase will be performed and the number of added lines decreased.

**Added tests:**
**_//job start tests with multiple online targets:_**

1. Starting a job with valid Package Install step and multiple devices
2. Starting a job with invalid Package Install step and multiple devices
3. Starting a job with valid Package Install and Bundle Start steps and multiple devices
4. Starting a job with invalid Package Install and Bundle Start steps and multiple devices

**_//job restart tests with multiple online targets_**

5. Restarting a job with valid Package Install Step, Multiple Devices And Step Index=0 For The First Time
6. Restarting a job with invalid Package Install Step, Multiple Devices And Step Index=0 For The First Time
7. Restarting a job with valid Package Install Step, Multiple Devices And Step Index=0 For The Second Time
8. Restarting a job with invalid Package Install Step, Multiple Devices And Step Index=0 For The Second Time
9. Restarting a job with valid Package Install and Bundle Start Steps, Multiple Devices And Step Index=0 For The First Time
10. Restarting a job with invalid Package Install and Bundle Start Steps, Multiple Devices And Step Index=0 For The First Time
11. Restarting a job with valid Package Install and Bundle Start Steps, Multiple Devices And Step Index=0 For The Second Time
12. Restarting a job with invalid Package Install and Bundle Start Steps, Multiple Devices And Step Index=0 For The Second Time

**Screenshots**
_None_

**Any side note on the changes made**
_None_

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hey @Coduz,
I have checked this branch and it looks OK. Can you please merge this PR when all tests complete?

**Comment by ct-ajovanovic** [comment]
Hi @Coduz 
I just rebased this to the latest develop, so the line count is now bellow 1K, after the merge of the linked PR.
Can you please merge this PR when the tests pass?

**Review by Coduz — APPROVED** [review]

**Pull request merged**
